great_expectations/cli/checkpoint.py

```diff
@@ -305,15 +305,40 @@ def checkpoint_run(checkpoint, directory):
         )
 
     if not results["success"]:
-        cli_message("Validation Failed!")
+        cli_message("Validation failed!")
         send_usage_message(context, event=usage_event, success=True)
+        print_validation_operator_results_details(results)
         sys.exit(1)
 
-    cli_message("Validation Succeeded!")
+    cli_message("Validation succeeded!")
     send_usage_message(context, event=usage_event, success=True)
+    print_validation_operator_results_details(results)
     sys.exit(0)
 
 
+def print_validation_operator_results_details(results: ValidationOperatorResult) -> None:
+    suite_name_width = 40
```

Comment:
Total nitpick but I'd name this `max_suite_name_width` or similar to indicate it's a limitation, not the length of an actual suite name, got me just a little confused about line 335.

Reply:
Good feedback - fixed!